packages/bff/src/routers/catalogRouter.ts

```diff
@@ -56,7 +56,27 @@ const catalogRouter = (
         return res.status(errorRes.status).json(errorRes).end();
       }
     })
-    .get("/producers/eservices", async (_req, res) => res.status(501).send())
+    .get("/producers/eservices", async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+      try {
+        const response = await catalogService.getProducerEServices(
+          req.query.q,
+          req.query.consumersIds,
+          req.query.offset,
+          req.query.limit,
+          ctx
+        );
+
+        return res.status(200).json(response).send();
+      } catch (error) {
+        const errorRes = makeApiProblem(
+          error,
+          bffGetCatalogErrorMapper,
+          ctx.logger
```

Comment:
You can include here the log as in Scala `"Error retrieving Producer EServices"`

Reply:
This [commit](https://github.com/pagopa/interop-be-monorepo/pull/834/commits/a4441b90d8e277c7114aa14c51a34ddf37a09939) introduce relative operative log